docs-src/concepts/what-is-a-list-filter.md

```diff
@@ -31,19 +31,12 @@ A List Filter contains [Search Attribute](/concepts/what-is-a-search-attribute)
   - **BETWEEN ... AND**
   - **IN**
   - **ORDER BY**
-  - **LIKE**
-
-- Wildcards ('\*', '%', etc.) are supported for string-type List Filter Search Attributes.
-  Use wildcards with the **LIKE** operator to query possible values:
-
-  ```bash
-  // Create a "ProductId" custom Search Attribute of type String
-  tctl admin cluster add-search-attributes --name ProductId --type String
-  // Match values that start with "book"
-  ProductId LIKE "book%"
-  // Match values that contain "favorite"
-  ProductId LIKE "%favorite%"
-  ```
+
+  The last operator (**ORDER BY**) is only supported with Elasticsearch as a Visibility store.
+  Custom Search Attributes of `Text` type cannot be used in **ORDER BY** clauses.
+
+- To search for a specific word, [set a custom Search Attribute of type `Text`](/app-dev-context/observability#custom-search-attributes) to a phrase that contains that word.
+  Any command run with that Search Attribute set to it will return any Workflows that contain that word.
```

Comment:
```suggestion
- To search for a specific word, set a [custom Search Attribute](/app-dev-context/observability#custom-search-attributes) of type `Text` to a phrase that contains that word.
  Any command run with that Search Attribute returns any Workflows that contain that word.
```
Two reasons for the change in the link text:
- For SEO, link text should match the target as closely as possible.
- Recent style change: Link text should not contain inline code (monospace).